pkg/data/domain/app/service.go

```diff
@@ -116,3 +116,10 @@ func (s *Service) getByName(ctx context.Context, namespace, name string) (Resour
 
 	return app, nil
 }
+
+// omitManagedFields removes managed fields to make YAML output easier to read.
+// With Kubernetes 1.21 we can use OmitManagedFieldsPrinter and remove this.
+func omitManagedFields(app *applicationv1alpha1.App) *applicationv1alpha1.App {
+	app.ManagedFields = nil
+	return app
+}
```

Comment:
OmitManagedFieldsPrinter got added in K8s 1.21. So we can remove this then.

https://github.com/kubernetes/cli-runtime/blob/v0.21.0/pkg/printers/managedfields.go#L26-L30

Reply:
👍🏻